FriendsAndPlaces/Functions/LogoutFunction.cs

```diff
@@ -9,22 +11,64 @@ namespace FriendsAndPlaces.Functions
 {
     public class LogoutFunction
     {
+        private readonly IDatabaseManager _databaseManager;
+
+        private const string _acceptHeaderApplicationJson = "application/json";
+        private const string _contentTypeHeaderApplicationJson = "application/json";
+
+        public LoginFunction(IDatabaseManager databaseManager)
+        {
+            _databaseManager = databaseManager;
+        }
+
         [FunctionName("Logout")]
-        public async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+        public async Task<IActionResult> Logout(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req,
```

Comment:
```suggestion
            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "logout")] HttpRequest req,
```
Dann müssen wir nicht auf das Auto-Mapping setzen.